registration/include/pcl/registration/ndt.h

```diff
@@ -168,13 +168,26 @@ class NormalDistributionsTransform : public Registration<PointSource, PointTarge
     outlier_ratio_ = outlier_ratio;
   }
 
+  /** \brief Get the registration alignment likelihood.
+   * \return transformation likelihood
+   */
+  inline double
+  getTransformationLikelihood() const
+  {
+    return trans_likelihood_;
+  }
+
   /** \brief Get the registration alignment probability.
    * \return transformation probability
    */
+  PCL_DEPRECATED(1,
+                 15,
+                 "The method `getTransformationProbability` is deprecated. Please use "
+                 "`getTransformationLikelihood` instead.")
   inline double
   getTransformationProbability() const
```

Comment:
Yes, I think that's a good solution with the union. Just two minor things: please deprecate this function for 1.16, not 1.15, and make it clear in the documentation that this is just a renaming, that the returned value is the same as before (getTransformationLikelihood vs getTransformationProbability)

Reply:
I agree, didn't think of an anonymous union on first try. By documentation, do you mean the deprecation message or the function comment \brief?

Reply:
Yep, the new deprecation message with the word "renamed" should do it